odxtools/templates/macros/printCompuMethod.xml.jinja2

```diff
@@ -110,7 +110,7 @@
 </PROG-CODE>
 {%- endmacro -%}
 
-{%- macro printCompuMethod(cm) -%}
+{%- macro printCompuMethod(cm) %}
```

Comment:
Why?

Reply:
the this screwed up the indentation of the XML output in some cases...